appveyor.yml

```diff
@@ -1,19 +1,13 @@
 version: 0.1.0.{build}
-skip_branch_with_pr: true
-image: Visual Studio 2017 Preview
-pull_requests:
-  do_not_increment_build_number: true
-skip_tags: true
-clone_depth: 5
-nuget:
```

Comment:
Are all these changes intentional/necessary?

Reply:
Oh you said it could be jiggered back. Okay ya that’d be cool, but otherwise lgtm